.github/workflows/e2e-embed.yml

```diff
@@ -0,0 +1,110 @@
+name: E2E test - embed
+on:
+  push:
+    branches: [ tests/ci-embed ]
+  pull_request_target: # So we can test on forks
+    branches:
+      - main
+    # Embed e2e - tests verify booking flow which is applicable to non-embed case also. So, don't ignore apps/web changes.
+    paths-ignore: 
+      - apps/api/**
+      - apps/console/**
+      - apps/docs/**
+      - apps/swagger/**
+      - apps/website/**
+      - public/**
```

Comment:
AFAIK we don't have a public folder on monorepo root? 🤔

Reply:
True. It should have beens apps/web/public

Reply:
Done